### PR TITLE
composeinfo: add docs for Compose class

### PR DESCRIPTION
**Pull request by ktdreyer**
Document what the `composeinfo.Compose` class is and how to use it.

### CODE REVIEW

File: productmd/composeinfo.py [hunk 1]
```diff
@@ -238,6 +244,11 @@ def cmp_label(label1, label2):
 
 
 class Compose(productmd.common.MetadataBase):
+    """
+    This class represents a the top level of metadata for a compose.
```
Comment (lubomir) [inline productmd/composeinfo.py]: Typo here? `a the`

File: productmd/composeinfo.py [hunk 2]
```diff
@@ -238,6 +244,11 @@ def cmp_label(label1, label2):
...
+
+    It provides the compose's ID, type, date, etc.
```
Comment (lubomir) [inline productmd/composeinfo.py]: Technically the listed data is under the `info` attribute. Maybe *It provides access to general information about the compose (ID, type, date, etc.) and structures with RPMs and images.* would be more accurate?

File: productmd/composeinfo.py (hunk 1 above)
Comment (ktdreyer) [reply productmd/composeinfo.py]: Whoops, thanks. I've amended and pushed with this fixed.

File: productmd/composeinfo.py (hunk 2 above)
Comment (ktdreyer) [reply productmd/composeinfo.py]: Sure, I've amended and pushed with this suggestion.

**Comment by lubomir** [comment]
Thank you!

**Pull request merged**